### DISCUSSION TITLE
Hacked merge burger

### DISCUSSION THREAD

**Discussion by hughes123b** [community/community#144387] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I thought someone hacked my account. I'm not gonna lie I have no idea what is going on. What is merge burger? Why have AI been stalking me and messing with my LIFE? ANSWERS PLS

**Comment by mecodeatlas** [comment]
Hi @hughes123b, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
